### ISSUE TITLE
Issue regarding connection between sub and master

### ISSUE THREAD

**Issue by ballgamer56** [QTDerg/switcher-roo#8]
It appears that the sub can view the master as being connected but the master side only sees the session ID screen with the message stuck at "connecting". tested on the same network and different network

**Comment by QTDerg** [comment]
Can you check now? I changed the relay server server to a different one

Sorry for late reply, I didn't get a notification for this thread

**Comment by ballgamer56** [comment]
Same issue, although the sub does, in fact, see the connected screen almost instantly now (tested with the same network only as of now)

But the Master side is still stuck on the connecting screen while the sub sees the master connected to them. Same as before. Will update when I get my friend to test it on a different network between each other

**Comment by QTDerg** [comment]
Could you check if this thing works correctly? - https://jmcker.github.io/Peer-to-Peer-Cue-System/

It's an example implementation of PeerJS library which I'm using. That way we'll know if the culprit is my app or something else

**Comment by ballgamer56** [comment]
Yeah, seems like the P2P system is the culprit, it works on the same device but not on different devices on the same network (and presumably devices on different networks too)

Also just checked, the system uses **port 443**, which is closed on my end (from basic check), so that's definitely a problem. Will update with you once I figured out how to fix it on my end

**Comment by QTDerg** [comment]
Yeah, a closed 443 port could cause issues. The master PeerJS server (the one which gives you these long IDs) which is currently used in the app forces HTTPS. I'm gonna add an option to switch to a different one

By the way, could you test the relay server (TURN) here? - https://icetest.atec-systems.com/

Set dropdown option to TURN
Domain - freeturn.net
Port - 3478
Username - free
Password - free
